addon/components/nypr-account-forms/login.js

```diff
@@ -37,15 +37,18 @@ export default Component.extend({
   },
   actions: {
     onSubmit() {
-      set(this, 'showLoginError', false);
+      set(this, 'loginError', null);
       return this.authenticate(get(this, 'fields.email'), get(this, 'fields.password'));
     },
     onFailure(e) {
       if (e) {
+        let errorCode = get(e, 'errors.code');
         if (get(e, 'errors.code') === 'AccountNotConfirmed') {
```

Comment:
this should probably be `errorCode`, right? this might read better as a switch statement too.